README.md

```diff
@@ -115,6 +115,22 @@ Options:
   --output       output directory path                  [string] [default: "./"]
 ```
 
+### Concerto Decorate
+
+The `decorate` command allows you to add decorators and vocabularies to your local CTO file from a set of local DCS and Vocab Command Set files
+
+```
+concerto decorate
+
+apply the decorators and vocabs to the target models from given list of dcs files and vocab files
+
+Options:
+  --model      The CTO file                                   
+  --voc        The List of vocab files                                             
+  --dcs        The List of decorator command set files                                            
+  --f          output format (cto or json), default: cto
```

Comment:
Does `--model` support multiple values? We should make sure that we mirror the other commands.

Can you also add some non-abbreviated option flags too, please? 
For example, `--vocabulary`, `--decoratorCommandsSet`/`--decorator`, ``--format`.

Adding an optional `--output` flag would also help us to mirror the other commands.

Reply:
`--model` only supports a single model file for this usecase. However, in the other commands, it is used for array of models (ex. `validate`).  In some places, an alias of `models` is provided.
Shall I change it to something else?

Reply:
Haven't really added support for writing to an external file. Will add it and provide `--output` flag too

Reply:
>  In some places, an alias of models is provided. Shall I change it to something else?

This should be considered a bug IMO. We should try to standardize the flags across commands, as far as possible. Can you get as close as you can for your new code, and open an issue for any other errant commands given that fixing them will be a breaking change, please?